python/dwave_qbsolv/qbsolv_binding.pyx

```diff
@@ -31,7 +31,7 @@ def run_qbsolv(Q, num_repeats=50, seed=17932241798878,  verbosity=-1,
     Arguments are described in the dimod wrapper.
 
     Returns:
-        (list, list): (samples, counts) where samples is a list of dicts,
+        (list, list, list): (samples, energies, counts) where samples is a list of dicts,
```

Comment:
eventually this should probably just return a numpy array then `Response.from_matrix(samples_array, ...)` but for now this is fine.